db/migrations/00001_user_tables.go

```diff
@@ -0,0 +1,48 @@
+package migrations
+
+import (
+	"database/sql"
+
+	"github.com/jinzhu/gorm"
+	"github.com/pressly/goose"
+
+	"github.com/iReflect/reflect-app/apps/user/models"
```

Comment:
Migrations should use frozen models and not directly point to current models file, as that may break this migration/future migrations.

Reply:
done